.github/workflows/Pipeclean_test.yml

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Run Pipeclean Test
+on:
+  workflow_dispatch:
+
+env:
+    AzureTestVmUsername: ${{ secrets.AZURE_TEST_VM_USER_NAME }}
```

Comment:
use uppercase with underscore for env file name  AzureTestVmUsername

Reply:
these env variable names are used in this way by the AzureTestLab.ps1 file that is also used in TC, that is the reason why this name was used